### DISCUSSION TITLE
Is ENCORE sufficiently general to be applied across different scientific domains?

### DISCUSSION THREAD

**Discussion by ahcvankampen** [EDS-Bioinformatics-Laboratory/ENCORE#14] [General]
ENCORE is agnostic to the type of computational project (e.g., statistical analysis, mathematical modelling), data, programming language, and ICT infrastructure (e.g., operating system and computer hardware). It should not depend on (project management) tools that are not freely available.

Although ENCORE  has been developed from the perspective of bioinformatics and computational modelling in the cellular and molecular biology field, it can be applied in other scientific disciplines to virtually any type of computational project.

However, so far, we never put this into practice but it is one of the next steps we aim to take.

If the approach is not sufficiently generic, then one way to go forward is the development of ENCORE specifications for different types of computational projects and/or different scientific domains by different specialized working groups. This might take into account standards and guidelines developed for specific fields.

**Comment by pdmoerland** [comment]
Regarding platforms that have already been developed for specific domains, during a talk at [BioSB2024](https://www.aanmelder.nl/biosb2024/program) (Julian Arts, Radboud UMC, Developing a Robust Machine Learning Pipeline for Predicting Corneal Cell States Using a Meta-Atlas) I heard for the first time of DagsHub (https://dagshub.com/) which is a "platform for AI and ML developers that lets you manage and collaborate on your data, models, experiments, alongside your code." GitHub repos can easily be connected to DagsHub, see https://dagshub.com/docs/integration_guide/github/.
